pathfinder/paizo/campaign_setting/inner_sea_combat/isc_equip_magic.lst

```diff
@@ -27,12 +27,12 @@ Lastwall Banner (Hordeline)						TYPE:Wondrous.Magic					COST:7500	WT:3		SOURCEP
 Lastwall Banner (Harchist)						TYPE:Wondrous.Magic					COST:15000	WT:3		SOURCEPAGE:p.61
 Lastwall Banner (Sunwall)						TYPE:Wondrous.Magic					COST:125000	WT:3		SOURCEPAGE:p.61
 Mantle of the crusader host						TYPE:Wondrous.Magic.Slot_Chest.Shirt		COST:95000	WT:0		SOURCEPAGE:p.62
-Pelt of the Ulfen Wolf							TYPE:Wondrous.Magic.Slot_Shoulders.Cape		COST:14800	WT:3		SOURCEPAGE:p.62
+Pelt of the Ulfen Wolf							TYPE:Wondrous.Magic.Slot_Shoulders.Cape		COST:14800	WT:3		SOURCEPAGE:p.62	NAMEISPI:YES
 Ring of the Adept Maneuvers						TYPE:Wondrous.Magic.Ring				COST:15000	WT:0		SOURCEPAGE:p.62
 Rod of Revolution								TYPE:Wondrous.Magic.Rod					COST:82000	WT:8		SOURCEPAGE:p.62
 Sabel Company Elite Saddle						TYPE:Wondrous.Magic.Saddle				COST:14000	WT:5		SOURCEPAGE:p.62
 Sandstorm Sandals								TYPE:Wondrous.Magic.Slot_Feet.Boot			COST:15000	WT:2		SOURCEPAGE:p.63
 Scabbard of the efreet							TYPE:Wondrous.Magic					COST:41000	WT:2		SOURCEPAGE:p.63
-seafaring stanchions							TYPE:Wondrous.Magic.Slot_Wrist.Bracer		COST:2400	WT:2		SOURCEPAGE:p.63
-summoner slayer eyes							TYPE:Wondrous.Magic.Slot_Eyes.Eyewear		COST:11000	WT:1		SOURCEPAGE:p.63
-tablet of the first law							TYPE:Wondrous.Magic					COST:3000	WT:5		SOURCEPAGE:p.63	SPROP:Read 10 minutes for +1 to Will saves from divine sources.
+Seafaring stanchions							TYPE:Wondrous.Magic.Slot_Wrist.Bracer		COST:2400	WT:2		SOURCEPAGE:p.63
+Summoner slayer syes							TYPE:Wondrous.Magic.Slot_Eyes.Eyewear		COST:11000	WT:1		SOURCEPAGE:p.63
```

Comment:
On 1/31/2015 8:11 PM, Andrew Maitland wrote:
>
> In 
> pathfinder/paizo/campaign_setting/inner_sea_combat/isc_equip_magic.lst 
> <https://github.com/PCGen/pcgen-newsources/pull/82#discussion_r23893825>:
>
> >  Ring of the Adept Maneuvers						TYPE:Wondrous.Magic.Ring				COST:15000	WT:0		SOURCEPAGE:p.62
> >  Rod of Revolution								TYPE:Wondrous.Magic.Rod					COST:82000	WT:8		SOURCEPAGE:p.62
> >  Sabel Company Elite Saddle						TYPE:Wondrous.Magic.Saddle				COST:14000	WT:5		SOURCEPAGE:p.62
> >  Sandstorm Sandals								TYPE:Wondrous.Magic.Slot_Feet.Boot			COST:15000	WT:2		SOURCEPAGE:p.63
> >  Scabbard of the efreet							TYPE:Wondrous.Magic					COST:41000	WT:2		SOURCEPAGE:p.63
> > -seafaring stanchions							TYPE:Wondrous.Magic.Slot_Wrist.Bracer		COST:2400	WT:2		SOURCEPAGE:p.63
> > -summoner slayer eyes							TYPE:Wondrous.Magic.Slot_Eyes.Eyewear		COST:11000	WT:1		SOURCEPAGE:p.63
> > -tablet of the first law							TYPE:Wondrous.Magic					COST:3000	WT:5		SOURCEPAGE:p.63	SPROP:Read 10 minutes for +1 to Will saves from divine sources.
> > +Seafaring stanchions							TYPE:Wondrous.Magic.Slot_Wrist.Bracer		COST:2400	WT:2		SOURCEPAGE:p.63
> > +Summoner slayer syes							TYPE:Wondrous.Magic.Slot_Eyes.Eyewear		COST:11000	WT:1		SOURCEPAGE:p.63
>
> You gonna fix the "syes" which was "eyes"?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/PCGen/pcgen-newsources/pull/82/files#r23893825>.
>
Fixed in my set do I need to create another PR or will it gram all the 
files at their current state and not when the original PR was created?